.changeset/new-dragons-bow.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
+---
+
+feature: Add 'cloudchamber curl command
```

Comment:
Please can we add a bit more content here. If a user is browsing the changelog and see this they would not know what this command does and/or is for.